db/old_migrations/patch-monitor-oauth-cc.sql

```diff
@@ -13,6 +13,9 @@ ALTER TABLE monitor
 ALTER TABLE monitor
     ADD oauth_scopes TEXT default null;
 
+ALTER TABLE monitor
+    ADD oauth_audience TEXT default null;
+
```

Comment:
please move this to a [proper migration](https://github.com/louislam/uptime-kuma/tree/master/db/knex_migrations) instead of changing an aleady applied migration